Scripts/CopyContextToField/CopyContextToField.yml

```diff
@@ -0,0 +1,50 @@
+args:
+- default: false
+  description: The context key to get the value from.
+  isArray: false
+  name: sourceContextKey
```

Comment:
our convention for arg names is snake case.
sourceContextKey => source_context_key

Reply:
Sure? I know for integrations, but most of the automation it's not, at least all the automations I wrote...